src/main/java/com/jmb/springfactory/service/task/TaskServiceImpl.java

```diff
@@ -29,122 +34,185 @@
 
 import com.jmb.springfactory.service.productionorder.ProductionOrderService;
 
+import lombok.val;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'lombok.val'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)